src/Settings/SettingsManager.php

```diff
@@ -87,30 +87,35 @@ public function getSettingsByName(array $domainNames, array $settingNames): arra
     {
         $settings = [[]];
 
+        $settingConfigModel = $this->providers['config']->getSettingsByName($domainNames, $settingNames);
```

Comment:
What are these changes to SettingsManager?

Reply:
Oups! Was not suppose to push this here :(